classes/ocaml.bbclass

```diff
@@ -22,13 +22,13 @@ OCAMLOPT_class-target="ocamlopt -cc '${CC} -fPIC'"
 OCAMLMKLIB_class-target="ocamlmklib -ldopt '--sysroot=${STAGING_DIR_TARGET} ${LDFLAGS}'"
```

Comment:
I was going to remark that this is double quotes outside and single quote inside which seems wrong.  But it's okay since btibake is expanding the variables inside the inner single quotes.

Reply:
Actually in this case is has to be this way. Otherwise the double-quotes used in `EXTRA_OEMAKE` will be ended by the double-quotes of `--sysroot...` statements when passed to `make`. I did not find a less hair-pulled way to get this right...

Reply:
Oh geez...  Thanks for tackling this.